app/frontend/helpers/wasmLoader.js

```diff
@@ -1,10 +1,8 @@
-import {Buffer} from 'buffer'
 // interface extendedWindow extends Window {
 //   wasm?: any;
 // }
 
 export default async function loadWasmModule() {
   // (window as extendedWindow).wasm = await import('@emurgo/js-chain-libs/js_chain_libs_bg.wasm')
   window.wasm = await import('@emurgo/js-chain-libs/js_chain_libs_bg.wasm')
-  window.Buffer = Buffer
```

Comment:
@PeterBenc I see you commited this line in January 2020 - do you know why we needed it?

https://github.com/vacuumlabs/adalite/blob/4975453b030991f8fc6856f76d41c0ffcf004dbd/app/frontend/helpers/wasmLoader.js#L9

Reply:
I remember trying something with it, I think it was a hotfix for something which was later solved properly. Its just a forgotten piece of code, to my knowledge the line is no longer useful for anything. I tested using the wasm library without this line and everything worked as expected.